File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p04012/p04012_num4_parsed.cpp
#include <iostream>
#include <map>

using namespace std;

int main() {
    string w;
    cin >> w;
    
    map<char, int> count;
    
    for (char c : w) {
        count[c]++;
    }
    
    bool isBeautiful = true;
    
    for (const auto& pair : count) {
        if (pair.second % 2 != 0) {
            isBeautiful = false;
            break;
        }
    }
    
    if (isBeautiful) {
        cout << "Yes";
    } else {
        cout << "No";
    }
    
    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p04012/p04012_num4_parsed.py
w = input().strip()

def is_beautiful(w):
    letters = [0] * 26
    
    for char in w:
        index = ord(char) - ord('a')
        letters[index] += 1
    
    for count in letters:
        if count % 2 != 0:
            return False
    
    return True

if is_beautiful(w):
    print("Yes")
else:
    print("No")